pkgs/development/java-modules/maven-minimal.nix

```diff
@@ -1,6 +1,6 @@
-{ stdenv, pkgs }:
+{ stdenv, lib, pkgs }:
```

Comment:
We should really remove pkgs here.

Reply:
Removed Java stuff from the PR completely.

Reply:
> We should really remove pkgs here.

Side note: I don't think it's wise introducing unrelated changes to these kinds of PRs.

Reply:
I did not mean in this PR. General in another one.